### ISSUE TITLE
Correlazione fra Documenti

### ISSUE THREAD

**Issue by Simsimus** [italia/design-scuole-wordpress-theme#365]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Proposta

Buongiorno,
attualmente un Documento può essere collegato ai Servizi ed alle Strutture, ma credo sia molto utile poterlo collegare anche ad altri Documenti, quindi vorrei suggerire, se fattibile, l'implementazione di tale possibilità.
Grazie e buon lavoro

### Motivazione e contesto

Ottimizzare la correlazione fra documenti.

**Comment by vcoopcsc** [comment]
Concordo, sarebbe davvero molto utile e seguirebbe la logica del collegamento tra servizio e servizio

**Comment by vcoopcsc** [comment]
Anche la nuova versione del tema non prevede questa possibilità
